src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  // eslint-disable-next-line no-debugger
+  if (!sourceString) {
+    return {};
+  }
+
+  const styleToObject = {};
+  const partsOfExpresions = sourceString.split(';');
+
+  partsOfExpresions.forEach((part) => {
+    const index = part.indexOf(':');
+
+    const name = part.substring(0, index).trim();
+    const value = part.substring(index + 1).trim();
+
+    if (name && value) {
+      styleToObject[name] = value;
+    }
+  });
```

Comment:
The 'forEach' callback function can be simplified by using array destructuring with the 'split' method directly in the parameter list. This would eliminate the need for the 'index' variable and manual substring operations.